README.md

```diff
@@ -1,32 +1,197 @@
-# model-spinal-rootlets
+# Spinal-Rootlets Segmentation on T2w Images
 
 ## 1) Project Overview
 
-The goal of this project is to develop a method based on the DL technique to segment and locate the spinal roots.
-The localization of these nerves is especially important for the electrode-based treatment of spinal cord injuries.
+The goal of this project is to develop a deep learning (DL) technique-based method to segment and locate the spinal
+roots.
 
-## 2) Work done
+## 2) Work Done
 
-After [reviewing](https://github.com/ivadomed/model-spinal-rootlets/issues/1#issue-1706345176) the available datasets,
-it was determined that none of them were suitable.
-Therefore, a new dataset was used, and each sample was labeled with a binary value representing the presence of
-spinal rootlets (0: False, 1: True).
+### A) Literature and Data Review
 
-The original dataset can be
-found [here](https://openneuro.org/datasets/ds004507/versions/1.0.1). [review](https://github.com/ivadomed/model-spinal-rootlets/issues/1#issue-1706345176)
-An initial labeling of 12 subjects was performed to train one fold of the nnUNetV2 3d_fullres model (refer
-to [issue#5](https://github.com/ivadomed/model-spinal-rootlets/issues/5)).
-The resulting model achieved a Dice score of approximately 0.52.
+No articles were found on this topic.
+
+After reviewing the available
+datasets ([issue#1](https://github.com/ivadomed/model-spinal-rootlets/issues/1#issue-1706345176)), it was determined
+that none of them were suitable.
+
+### B) Dataset Creation
+
+Initially, the plan was to test with a binary label (0: no rootlet, 1: rootlet). In the following text, the datasets
+will be numbered as D1, D2, etc., and the models as M1, M2, etc.
+
+The original dataset (D0a) can be found [here](https://openneuro.org/datasets/ds004507/versions/1.0.1). An initial labeling of
```

Comment:
Shouldn't be `D1a`?

```suggestion
The original dataset (D1a) can be found [here](https://openneuro.org/datasets/ds004507/versions/1.0.1). An initial labeling of
```

Reply:
I planned to name the raw dataset D0 so D0a for the open neuro one and D0b for the spine-generic. 
In the original openneuro dataset there are 10 subjects with 3 sessions but I never use more than 10 subjects and 2 sessions per subject.